packages/react-widgets/src/less/mixins.less

```diff
@@ -107,18 +107,13 @@
 
 .height-calc(@total, @minus) {
   height: ~"calc(@{total} - @{minus})";
-
-  /** ie8 fallback **/
-  margin-top: ~"-@{minus}\9";
-  height: ~"@{total}\9";
-  /** --- **/
```

Comment:
why are you removing this?

Reply:
Because less produces this:

```css
margin-top: -6px 9;
height: 15px 9;
```

Note: values are not specific.

Reply:
Can this one go into the current branch since less is generating invalid CSS?

Reply:
we should fix the hack here to `/9`